lib/reference/properties.js

```diff
@@ -311,55 +320,70 @@ const longhandSubPropertiesOfShorthandProperties = new Map([
 			'mask-origin',
 			'mask-clip',
 			'mask-composite',
-		]),
+		],
 	],
 	[
 		'outline',
-		new Set([
+		[
 			// prettier-ignore
 			'outline-color',
 			'outline-style',
 			'outline-width',
-		]),
+		],
 	],
 	[
 		'padding',
-		new Set([
+		[
 			// prettier-ignore
 			'padding-top',
+			'padding-right',
 			'padding-bottom',
 			'padding-left',
-			'padding-right',
-		]),
+		],
 	],
 	[
 		'text-decoration',
-		new Set([
+		[
 			// prettier-ignore
 			'text-decoration-color',
 			'text-decoration-style',
 			'text-decoration-line',
```

Comment:
```suggestion
			'text-decoration-line',
			'text-decoration-style',
			'text-decoration-color',
			'text-decoration-thickness',
```

Reply:
Thanks for catching the wrong ordering - fixed it!

Where are you getting `text-decoration-thickness` from? The [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration#formal_syntax) seem to indicate that the syntax is

```
text-decoration = 
  <'text-decoration-line'>   ||
  <'text-decoration-style'>  ||
  <'text-decoration-color'>  

```

Reply:
> Note: The property used to be called text-decoration-width, but was updated in 2019 to text-decoration-thickness.

Should we support both `text-decoration-width` and `text-decoration-thickness`?

> Where are you getting text-decoration-thickness from?

> This property, which is a [sub-property](https://w3c.github.io/csswg-drafts/css-cascade-5/#longhand) of the [text-decoration](https://w3c.github.io/csswg-drafts/css-text-decor-4/#propdef-text-decoration) shorthand…

ref: https://w3c.github.io/csswg-drafts/css-text-decor-4/#text-decoration-width-property

Reply:
Ah! Thanks for pointing me to the spec; perhaps the MDN docs need to be updated?

The [spec ref for the shorthand](https://w3c.github.io/csswg-drafts/css-text-decor-4/#text-decoration-property) indicates that the syntax is:

[<'text-decoration-line'>](https://w3c.github.io/csswg-drafts/css-text-decor-4/#propdef-text-decoration-line) [||](https://w3c.github.io/csswg-drafts/css-values-4/#comb-any) [<'text-decoration-thickness'>](https://w3c.github.io/csswg-drafts/css-text-decor-4/#propdef-text-decoration-thickness) || [<'text-decoration-style'>](https://w3c.github.io/csswg-drafts/css-text-decor-4/#propdef-text-decoration-style) || [<'text-decoration-color'>](https://w3c.github.io/csswg-drafts/css-text-decor-4/#propdef-text-decoration-color).

I can definitely add thickness, but that would also change the existing behaviour of this rule. What do we think?

Reply:
I guess you could put that and the inset thing in the same enhancement issue.

Reply:
Can you put a TODO comment so that it's not forgotten?